app/models/responsible_body.rb

```diff
@@ -261,21 +260,20 @@ def trust?
     type == 'Trust'
   end
 
-  def vcap_active?
-    vcap_feature_flag? && orders_managed_centrally?
-  end
-
   def vcap_schools
-    return School.none unless vcap_active?
-
-    schools
-      .excluding_la_funded_provisions
-      .responsible_body_will_order_devices
-      .or(schools.who_will_order_devices_not_set)
+    return School.none unless vcap_feature_flag?
+
+    not_laf_schools = schools.excluding_la_funded_provisions
+    not_laf_schools_centrally_managed = not_laf_schools.responsible_body_will_order_devices
+    if orders_managed_centrally?
+      not_laf_schools_centrally_managed.or(not_laf_schools.who_will_order_devices_not_set)
+    else
+      not_laf_schools_centrally_managed
+    end
```

Comment:
This is difficult to read. Maybe it could be pushed to a scope in the `School` model?

Reply:
What about this?
 ```
  def vcap_schools
    return School.none unless vcap_feature_flag?
    return schools.excluding_la_funded_provisions.school_not_set_to_order_devices if orders_managed_centrally?

    schools.excluding_la_funded_provisions.responsible_body_will_order_devices
  end
````

Reply:
I would keep this into the ResponsibleBody model as it has to do with the internal logic on how a vcap works. Vcaps live in the ResponsibleBody now, not VirtualCapPool or School anymore.

Reply:
Agreed